spec/validation/meta_yml_spec.rb

```diff
@@ -1,103 +1,140 @@
 # frozen_string_literal: true
 require "spec_helper"
+require "pry"
 
-module HathiTrust
-  module Validation
+include HathiTrust
```

Comment:
It seemed odd to me to be throwing tests into the actual module under test. The `include` there is the same as any `include` at the top level: it brings those things into the top-level namespace so you can refer to them without their prefix. I think the only functional reason I did this is so I could have this:

```
describe Validation::MetaYml do
  let(:mocked_sip) { SIP::SIP.new("") }
```
rather than needing to include a shared context at each smaller scope.